alphazero/model.py

```diff
@@ -0,0 +1,101 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class ResidualBlock(nn.Module):
+    def __init__(self, in_channels, out_channels):
+        super(ResidualBlock, self).__init__()
+        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size=3, stride=1)
+        self.batch_norm = nn.BatchNorm2d(out_channels)
+        self.relu = nn.ReLU(inplace=true)
```

Comment:
Can you make the batch norm optional? As well as maybe a param for instead of ReLU something like LeakyReLU or something else.

Also, if you're declaring the ReLU to be inplace here, then I think later on (lines 24, 32) you're misusing it, since you're returning the value.

Reply:
I'll work on the params for batch norm/activations later, but I don't really get what you mean by returning the value? The inplace just changes the original tensor without allocating memory for a new tensor. I'll see if it works when I run tests.

Reply:
yeah, exactly. later on you act as if you're getting a new tensor as a result from the relu, but you're not.

Reply:
changed inplace to false. Will start running tests later tonight or tomorrow.